drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -477,6 +483,47 @@ static int mlx5_vdpa_release_rx_steer(struct vdpa_priv *priv)
 	return 0;
 }
 
+static
+int mlx5_vdpa_create_tis(struct vdpa_priv *priv)
+{
+	uint32_t in[MLX5_ST_SZ_DW(create_tis_in)] = {0};
+	uint32_t out[MLX5_ST_SZ_DW(create_tis_out)] = {0};
+	struct mlx5_vdpa_tx_steer_info *tis = NULL;
+
+	tis = rte_zmalloc("tis", sizeof(*tis), RTE_CACHE_LINE_SIZE);
+	MLX5_SET(create_tis_in, in, opcode, MLX5_CMD_OP_CREATE_TIS);
```

Comment:
was the TIS creation successful? which transport domain you used?

Reply:
It was, without any transport domain there is just no enforcement on LB traffic